tasks_management/services.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from django.db import transaction
+
+from core.services import create_or_update_core_user, BaseService
+from core.services.utils import check_authentication, output_exception, model_representation, output_result_success
+from tasks_management.models import TaskGroup, TaskExecutor
+from tasks_management.validation import TaskGroupValidation, TaskExecutorValidation
+
+logger = logging.getLogger(__name__)
+
+
+class TaskGroupService:
```

Comment:
Is there reason why TaskGroup is not base service? 
If it's not possible we could still point that certain interfaces of the base services are implemented.